multi-educator/src/Dscp/Educator/Web/Educator/Queries.hs

```diff
@@ -0,0 +1,164 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Dscp.Educator.Web.Educator.Queries
+    ( module Dscp.Educator.Web.Educator.Queries
+    ) where
+
+import Control.Lens (from, mapping, traversed, _Just)
+import Data.List (groupBy)
+import Data.Time.Clock (getCurrentTime)
+import Database.SQLite.Simple (Only (..), Query)
+import Servant (err501)
+import Text.InterpolatedString.Perl6 (q)
+
+import Dscp.Core
+import Dscp.Crypto
+import Dscp.DB.SQLite
+import Dscp.Educator.Web.Educator.Types
+import Dscp.Educator.Web.Types
+import Dscp.Util
+
+educatorRemoveStudent
+    :: MonadEducatorWebQuery m
+    => Student -> DBT t 'Writing m ()
+educatorRemoveStudent student = do
+    -- TODO [DSCP-176]: Proper implementation of this method may require
+    -- fundemental rethinking of our database scheme and rewriting many code.
+    -- Should be done soon though.
+    _ <- throwM err501
```

Comment:
The DSCP-176 is "Done". Wow. I assume, that thing exists in the code you're copied. @flyingleafe, should we reopen DSCP-176, instead of throwing internal errors?